app/src/main/java/com/github/se/eventradar/ui/home/Home.kt

```diff
@@ -0,0 +1,190 @@
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.github.se.eventradar.R
+import com.github.se.eventradar.model.event.Event
+import com.github.se.eventradar.model.event.EventCategory
+import java.time.LocalDate
+import java.time.LocalTime
+
+@Composable
+fun HomeScreen() {
+  val mockEvents =
+      listOf(
+          Event(
+              "1",
+              "User1",
+              "Jazz Concert",
+              "An evening with smooth jazz.",
+              LocalDate.of(2024, 4, 12),
+              LocalTime.of(19, 30),
+              EventCategory.MUSIC),
+          Event(
+              "2",
+              "User2",
+              "Soccer Game",
+              "Cheer for your local soccer team!",
+              LocalDate.of(2024, 4, 18),
+              LocalTime.of(15, 45),
+              EventCategory.SPORTS),
+          Event(
+              "3",
+              "User3",
+              "Tech Conference",
+              "The latest in tech innovation.",
+              LocalDate.of(2024, 5, 24),
+              LocalTime.of(10, 0),
+              EventCategory.CONFERENCE))
+
+  var selectedTabIndex by remember { mutableIntStateOf(0) }
+
+  Column(modifier = Modifier.fillMaxSize()) {
+    Spacer(modifier = Modifier.height(50.dp))
+    Row(
+        modifier = Modifier.fillMaxWidth().padding(start = 32.dp),
+        verticalAlignment = Alignment.CenterVertically) {
+          Image(
+              painter = painterResource(id = R.drawable.event_logo),
+              contentDescription = "Event Radar Logo",
+              modifier = Modifier.size(width = 184.35.dp, height = 46.99.dp))
+        }
+    Spacer(modifier = Modifier.height(16.dp))
+    TabRow(
+        selectedTabIndex = selectedTabIndex,
+        modifier = Modifier.fillMaxWidth().height(96.dp).padding(32.dp),
+        contentColor = MaterialTheme.colorScheme.primary) {
+          Tab(
+              selected = selectedTabIndex == 0,
+              onClick = { selectedTabIndex = 0 },
+          ) {
+            Text(
+                text = "Browse",
+                style =
+                    TextStyle(
+                        fontSize = 19.sp,
+                        lineHeight = 17.sp,
+                        fontFamily = FontFamily(Font(R.font.roboto)),
+                        fontWeight = FontWeight(500),
+                        color = MaterialTheme.colorScheme.onPrimaryContainer,
+                        textAlign = TextAlign.Center,
+                        letterSpacing = 0.25.sp,
+                    ))
```

Comment:
To give the "Browse" text a little bit more breathing room I would add some padding here.
```suggestion
                    ))
                Spacer(modifier = Modifier.height(8.dp))
```
Note that this suggestion is based on the current implementation but in the future it should be done with a constraint layout.